doc/configuration/telegram.md

```diff
@@ -23,6 +23,20 @@ you can set `TELEGRAM_AUTH_TOKEN` in the `.env.local` file, e.g.,
 TELEGRAM_BOT_AUTH_TOKEN=itsme55667788
 ```
 
+The alerting strategies will use Telegram bot notification without further configuration. You can check the [pricealert
+yaml file](../../config/pricealert-tg.yaml) in the `config/` directory for example.
+
+If you want the order submitting/filling notification, add the following to your `bbgo.yaml`:
+
+```yaml
+notifications:
+  routing:
+    trade: "$symbol"
+    order: "$symbol"
+    submitOrder: "$session"
+    pnL: "bbgo-pnl"
+```
```

Comment:
this is not supported for telegram.

Reply:
but if i add this section in yaml, i can get order notification from tg bot

Reply:
oh ok